src/Provider/Doctrine/Configuration.php

```diff
@@ -30,16 +28,6 @@ class Configuration implements ConfigurationInterface
 
     private ?array $entities = null;
 
-    /**
-     * @var array<StorageServiceInterface>
-     */
-    private array $storageServices = [];
-
-    /**
-     * @var array<AuditingServiceInterface>
-     */
-    private array $auditingServices = [];
-
```

Comment:
This should also be removed from `Configuration::configureOptions()`

Reply:
done